tokio-util/src/lib.rs

```diff
@@ -14,6 +14,10 @@
 #![cfg_attr(docsrs, feature(doc_cfg))]
 
 //! Utilities for working with Tokio.
+//!
+//! This crate contains items that are not semantically versioned
+//! with the core `tokio` crate. Things within this crate are subject
+//! to change.
```

Comment:
Nits: 

1. Link to the main Tokio crate on crates.io?
2. _How_ are things subject to change in this crate?

Reply:
Tokio has a clear roadmap and versioning scheme while this may never hit 1.0. So things are subject to change in the future.

Reply:
When initially reading this, I was unsure if it was saying that the crate doesn't respect semver wrt. breaking changes.

Reply:
Does this make more sense in the update?

Reply:
See my suggested change.